it/controllers/IndexControllerISpec.scala

```diff
@@ -96,6 +96,45 @@ class IndexControllerISpec extends IntegrationSpecCommonBase {
     breathingSpace = None
   )
 
+  val getPenaltyDetailsPayloadWithOverThreshold = GetPenaltyDetails(
```

Comment:
This scenario doesn't give accurate data, this is a point applied after the threshold so ideally there needs to be other points